frontend/src/metabase/admin/people/containers/GroupDetailApp.tsx

```diff
@@ -19,14 +22,19 @@ export const GroupDetailApp = (props: any) => {
     getGroupReq.isLoading ?? membershipsByUserReq.isLoading ?? !currentUser;
 
   return (
-    <LoadingAndErrorWrapper error={error} loading={isLoading}>
-      {currentUser && (
-        <GroupDetail
-          membershipsByUser={membershipsByUserReq.data ?? {}}
-          group={getGroupReq.data!}
-          currentUser={currentUser}
-        />
-      )}
-    </LoadingAndErrorWrapper>
+    <SettingsPageWrapper
+      title={t`Groups`}
+      description={t`You can use groups to control your users' access to your data. Put users in groups and then go to the Permissions section to control each group's access. The Administrators and All Users groups are special default groups that can't be removed.`}
```

Comment:
I think we might want to put a max-width on the description text on the wrapper here. This description being as long as it is on this page feels a bit funky. Non blocking on this PR I think but something to follow up on.

Reply:
I think I'd actually recommend we put the description at the top of the main group listing content block instead for now. I think that'll more naturally make sense rather than having it repeated at the top of every page.